core/modules/ckeditor/ckeditor.install

```diff
@@ -13,20 +13,32 @@ function ckeditor_requirements($phase) {
 
   // Check if CKEditor is not enabled on any formats.
   if ($phase == 'runtime') {
-    $ckeditor_found = FALSE;
+    $ckeditor_formats = array();
     $formats = filter_formats();
     foreach ($formats as $format) {
       if (isset($format->editor) && $format->editor === 'ckeditor') {
-        $ckeditor_found = TRUE;
+        $ckeditor_formats[] = $format->name;
       }
     }
-    if (!$ckeditor_found) {
-      $requirements['ckeditor_not_enabled'] = array(
-        'title' => $t('CKEditor'),
-        'value' => $t('CKEditor is installed but not enabled on any formats. Visit the <a href="!url">text editors and formats configuration</a> to enable CKEditor on one or more formats.', array('!url' => url('admin/config/content/formats'))),
-        'severity' => REQUIREMENT_WARNING
-      );
+
+    if (empty($ckeditor_formats)) {
+      $description = $t('CKEditor is installed but not enabled for any text format.');
+    }
+    else {
+      $description = $t('CKEditor is installed and enabled for the following text formats:');
+      $description .= theme('item_list', array('items' => $ckeditor_formats));
     }
+
+    if (user_access('administer filters')) {
+      $description .= ' ' . $t('To configure text formats, visit the <a href="!url">Text editors and formats</a> configuration page.', array('!url' => url('admin/config/content/formats')));
```

Comment:
What's the reason for using `!url` here (as opposed to `@url`)? The former is more dangerous as no sanitisation is performed. So if `@url` would work just as well, I'd recommend that instead...

```suggestion
      $description .= ' ' . $t('To configure text formats, visit the <a href="@url">Text editors and formats</a> configuration page.', array('@url' => url('admin/config/content/formats')));
```